template-compose/detekt-config.yml

```diff
@@ -202,8 +203,8 @@ naming:
     packagePattern: '^[a-z]+(\.[a-z][a-z0-9]*)*$'
   TopLevelPropertyNaming:
     active: true
-    constantPattern: '[A-Z][_A-Z0-9]*'
-    propertyPattern: '[a-z][A-Za-z\d]*'
+    constantPattern: '[A-Z][_A-Za-z0-9]*'
+    propertyPattern: '[A-Za-z][_A-Za-z0-9]*'
```

Comment:
Why do we need to update this? Could you please suggest me as I didn't see it was suggested in the [document](https://detekt.dev/docs/introduction/compose/#toplevelpropertynaming). 🙏🏻

Reply:
@lydiasama As we have discussed on this [Thread](https://nimble-co.slack.com/archives/G41UCTGHL/p1668056608803069), We don't wrap compose for `Color.kt` or `Dimension.kt` in `Object` class. That means when we remove `Object`, then we will get the warning from `Detekt`:

![Screen Shot 2565-12-12 at 17 35 51](https://user-images.githubusercontent.com/28002315/207024372-bbe4e89a-50a5-48e7-b583-33c6f37039a8.png)
![Screen Shot 2565-12-12 at 17 36 16](https://user-images.githubusercontent.com/28002315/207024404-2a87ac6d-3612-4690-a5e9-5beeadfd7af3.png)


Regarding this, we need to update the `pattern` for `propertyPattern`. For more [ref](https://detekt.dev/docs/rules/naming/#toplevelpropertynaming) 🙏

Reply:
@Wadeewee BTW, we don't declare the `snake_case` for the properties, right? Shall we get rid of `_` in pattern?
```suggestion
    propertyPattern: '[A-Za-z][A-Za-z0-9]*'
```

Reply:
Oh yeah 😅  your right, @lydiasama 👍 I don't beware of `_` when I copied from the [doc](https://detekt.dev/docs/rules/naming/#toplevelpropertynaming)  Thank to spotting this 🙏